services/QuillLMS/app/serializers/diagnostic_insights/student_learning_sequence_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module DiagnosticInsights
+  class StudentLearningSequenceSerializer < ActiveModel::Serializer
+    attributes :id, :created_at, :initial_activity_id, :name, :user_id
+
+    has_many :student_learning_sequence_activities do
+      serialized_activities = object.student_learning_sequence_activities.map { |activity| StudentLearningSequenceActivitySerializer.new(activity).as_json }
+      serialized_activities.group_by { |activity| activity[:activity_type] }.symbolize_keys
+    end
+
+    def name = object.user.name
```

Comment:
Take or leave but I kind of want this to be `user_name`? When I first read it I was like, oh, learning sequences have names?